internal/pkggraph/tree.go

```diff
@@ -1,86 +1,268 @@
 package pkggraph
 
 import (
-	"path"
+	"fmt"
+	"os/exec"
+	"path/filepath"
 	"sort"
 	"strings"
+
+	"golang.org/x/mod/module"
+	"golang.org/x/tools/go/packages"
+	"golang.org/x/tools/go/vcs"
 )
 
+func (g *Graph) Tree() (*Tree, error) {
```

Comment:
At this point, I would probably move the `Tree` into a separate package `pkgtree`. This should help clean up the naming as well.

Reply:
Agree. Thanks for the suggestion. I'll work on that.